pkg/tests/smoke/testdata/Bob/test.gpt

```diff
@@ -7,4 +7,4 @@ name: bob
 description: I'm Bob, a friendly guy.
 args: question: The question to ask Bob.
 
-When asked how I am doing, respond with exactly "Thanks for asking "${question}", I'm doing great fellow friendly AI tool!"
+When asked how I am doing, respond with exactly "Thanks for asking "${QUESTION}", I'm doing great fellow friendly AI tool!"
```

Comment:
I see this was merged already but as a point of interest, this is not really an env var - this tool is not a 'code tool', so this interpolation syntax is really just there as a hint for the llm.